Bottom: 4c3a27d7e02497fc1d95f73365a8036808d844e3
Top:    6c7033512e616d587638246a8e7ffdba7815fbfe
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:15 +0000

Bump circle approximation detail level for gerber export


---

diff --git a/src/polygon.h b/src/polygon.h
index 58634f7..90d0d26 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 24 //20 //8 //40
+#define POLY_CIRC_SEGS 40 //24 //20 //8 //40
 #define POLY_CIRC_SEGS_D ((double)POLY_CIRC_SEGS)
 
 #if 0
